tests/requirements-libraries.txt

```diff
@@ -51,28 +52,23 @@ ipython==7.16.1; python_version == '3.6'  # pyup: ignore
 ipython==7.9.0; python_version == '3.5'  # pyup: ignore
 
 # pandas 1.0.0 dropped Python 3.5 support.
-pandas==1.1.3; python_version > '3.5'
+# Not yet Python 3.9 compatible because of Numpy
+pandas==1.1.3; python_version > '3.5' and python_version < '3.9'
```

Comment:
I understand that numpy ist not yet available for 3.9- But this change inhibits pandas to be tested again a soon ans it is available for Python 3.9. So we need to what pandas/numpy for py3.9 support. Anyhow, this should not be necessay, since if there is no pandas for 3.9, tests are skiped.